src/main/groovy/com/sequenceiq/cloudbreak/client/CloudbreakClient.groovy

```diff
@@ -130,6 +133,19 @@ class CloudbreakClient {
         return response?.data?.id
     }
 
+    def String postRecipe(String recipe, Boolean publicInAccount) throws Exception {
```

Comment:
It is exactly the case, recipes are read from files just like blueprints in the shell. I'll update the client in a later PR to include a more sophisticated method to create recipes.